avm/res/cache/redis/tests/e2e/passive-geo-replication/main.test.bicep

```diff
@@ -0,0 +1,157 @@
+targetScope = 'subscription'
+
+metadata name = 'Passive Geo-Replicated Redis Cache'
+metadata description = 'This instance deploys the module with geo-replication enabled.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-cache.redis-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'crpgeo'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+// ============ //
+// Dependencies //
+// ============ //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: resourceLocation
+}
+
+module nestedDependencies1 'dependencies1.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, resourceLocation)}-nestedDependencies1'
+  params: {
+    location: resourceLocation
+    managedIdentityName: 'dep-${namePrefix}-msi-ds-${serviceShort}'
+    pairedRegionScriptName: 'dep-${namePrefix}-ds-${serviceShort}'
+  }
+}
+
+module nestedDependencies2 'dependencies2.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, resourceLocation)}-nestedDependencies2'
+  params: {
+    managedIdentityName: 'dep-${namePrefix}-msi-sec-${serviceShort}'
+    location: nestedDependencies1.outputs.pairedRegionName
+    redisName: 'dep-${namePrefix}-redis-sec-${serviceShort}'
+  }
+}
+
+module nestedDependencies3 'dependencies3.bicep' = {
```

Comment:
How come this can't be part of `nestedDependencies1` or `nestedDependencies2`? Can't see a clear dependency

Reply:
Fixed. This was left over when I was attempting to test using private cache's deployed in VNET's.